### PR TITLE
Connecting Basic4Android (B4A) Application to PHP and MySQL Database

### PR DESCRIPTION
**Pull request by tambastar**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by tambastar** [comment]
this fixes #1251

**Comment by hectorkambow** [comment]
@Tsanguu  will be reviewing this article -  🚀

**Comment by hectorkambow** [comment]
plagiarism check done @Tsanguu

**Comment by Tsanguu** [comment]
Working on it @hectorkambow

**Review by Tsanguu — COMMENTED** [review]
@tambastar Great tutorial. The content is well detailed. There are a few issues I have pointed on that can improve this tutorial. Check the recommendations. Work on them so that we can move to the next step.

**Comment by tambastar** [comment]
Thank you @Tsanguu. all changes suggested have been worked on accordingly

Regards

**Comment by tambastar** [comment]
@Tsanguu  sorted

**Review by Tsanguu — CHANGES_REQUESTED** [review]
There are some areas I suggest you work on to improve the tutorial.

The B4A code provided is running. There is an issue; the B4A app cannot connect to the server. To make it work, you have to edit the manifest file. Find the solution to this issue on the [B4A forum](https://www.b4x.com/android/forum/threads/android-usescleartexttraffic-true-need-help.106915/.
).

Once the server's connection is made, the app gives positive feedback, yet no data is inserted in the database.
I suggest that you verify that the code or the final product is working as expected.

File: Connect-Basic4Android-BAndroid-development-too- to-MySQL-Database-Using-PHP/index.md [hunk 1]
```diff
@@ -0,0 +1,623 @@
...
+
+### Prerequisites
+
+* Have prior SQL, PHP, and B4A programming language.
+* B4A application installation on a Windows computer. If not installed, follow the instruction in this [article](https://www.section.io/introduction-to-android-app-development-basic4android-b4a-part1/) on how to install and configure.
+* PHP and MySQL development environment: In this case, we will use Xampp.
+
+### Installing B4A
+
+* Kindly follow this [Tutorial](https://www.section.io/introduction-to-android-app-development-basic4android-b4a-part1).
```
[the remark this answers is not in the document]
Comment (tambastar) [reply Connect-Basic4Android-BAndroid-development-too- to-MySQL-Database-Using-PHP/index.md:17]: this link is invalid. Kindly check on it.
i have renamed to one that is redirecting correctly

File: Connect-Basic4Android-BAndroid-development-too- to-MySQL-Database-Using-PHP/index.md [hunk 2]
```diff
@@ -0,0 +1,623 @@
...
+
+* Select the folder where to install  Xampp, and click Next.
+
+![Installation folder](/engineering-education/Connect-Basic4Android-BAndroid-development-too--to-MySQL-Database-Using-PHP/xampp-third-folder-selection.png)
+
+* Select the language, and click next.
+
+![Language](/engineering-education/Connect-Basic4Android-BAndroid-development-too--to-MySQL-Database-Using-PHP/xampp-fourth-language-selection.png)
+
+* Click next for the below windows.
```
[the remark this answers is not in the document]
Comment (tambastar) [reply Connect-Basic4Android-BAndroid-development-too- to-MySQL-Database-Using-PHP/index.md:38]: done

**Comment by tambastar** [comment]
I have done the changes requested.
Kindly put right URL for the app to communicate
@Tsanguu

**Review by Tsanguu — APPROVED** [review]

**Comment by Tsanguu** [comment]
@hectorkambow This article is ready for final review.

**Comment by WanjaMIKE** [comment]
@Tsanguu I added for you the ready for final review label

**Comment by hectorkambow** [comment]
@tambastar  

Please take note of the changes made during the final review - there were some changes to the intro and I found some place a bit tough to go through in the article as well. 

cc @Tsanguu  Please also take keen note of my changes to better improve any future revisions 🚀

**Pull request merged**